machineconfig/master/spec/config/storage/files/keepalived.conf.template L2V0Yy9rZWVwYWxpdmVkL2tlZXBhbGl2ZWQuY29uZi50ZW1wbGF0ZQo=

```diff
@@ -0,0 +1,22 @@
+vrrp_script chk_ocp {
+    script "/etc/keepalived/check_ocp_api.sh 127.0.0.1"
```

Comment:
The parameter being passed into this script is being ignored.

Reply:
Very true. Sorry about that.